react/components/EXPERIMENTAL_Table/hooks/useTableBulkActions.tsx

```diff
@@ -129,7 +130,20 @@ const useTableBulkActions = ({
     [bulkState.selectedRows, bulkState.allLinesSelected]
   )
 
+  const isRowSelected = useCallback(
+    (row: unknown) => {
+      return (
+        bulkState &&
+        bulkState.selectedRows.some(
+          selectedRow => selectedRow[unicityKey] === row[unicityKey]
+        )
+      )
```

Comment:
Here. It makes sense to use the `condition && method()` because it's returning to someone who will actually use it's value